pyproject.toml

```diff
@@ -0,0 +1,68 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dask-image"
+authors = [{name="dask-image contributors"}]
+description = "Distributed image processing"
+keywords = ["dask-image", "dask", "image"]
+readme = "README.md"
+license = { file="LICENSE.txt" }
+dynamic = ["version"]
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+]
+dependencies = [
+    "dask[array] >=2021.10.0",
+    "numpy >=1.11.3",
+    "scipy >=0.19.1",
+    "pims >=0.4.1",
+    "tifffile >=2018.10.18",
```

Comment:
In the meantime the dependencies have been updated in `setup.py`:
- "dask[array] >=2023.2.0",
- "dask[dataframe] >=2023.2.0",
- "pandas >=2.0.0",